src/query.ts

```diff
@@ -265,19 +267,24 @@ export function stringifyEncodingQueryFieldDef(encQ: EncodingQuery): string {
   if (encQ.scale && !isEnumSpec(encQ.scale)) {
 
       if (encQ.scale && !isEnumSpec(encQ.scale)) {
-      var scaleParams = {};
-
-      if (encQ.scale['type']) {
-        scaleParams = {type: encQ.scale['type']};
-      }
-      // TODO: push other scale properties to scaleParams.
-
-      if (keys(scaleParams).length > 0) {
-        params.push({
-          key: 'scale',
-          value: JSON.stringify(scaleParams)
-        });
-      }
+
+      // // TODO: push other scale properties to scaleParams (add it into the array).
+
+        var scaleParams = ['type', 'zero']
```

Comment:
This looks correct.  However, we can reduce one location we have to update everytime we add a parameter by following this [suggestion](https://github.com/uwdata/compassql/pull/113/files#r70013361).